src/mlpack/methods/ann/layer/adaptive_max_pooling.hpp

```diff
@@ -51,7 +51,6 @@ class AdaptiveMaxPooling
    * @param outputShape A two-value tuple indicating width and height of the output.
    */
   AdaptiveMaxPooling(const std::tuple<size_t, size_t>& outputShape);
```

Comment:
```suggestion
  AdaptiveMaxPooling(const std::tuple<size_t, size_t>& outputShape);

```
Even though the whitespaces here were unnecessary, the newline is required.